src/components/pages/Project.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+    <div class="gradientBackground">
+        <div class="titleHeader text-center">
+            <h1 class="display-3 hidden-sm-and-down font-weight-thin white--text ">
+                {{project.type === 'donate' ? 'Spendenseite' : 'Betriebsseite'}}
+            </h1>
+            <h1 class="display-1 hidden-md-and-up font-weight-thin white--text ">
+                {{project.type === 'donate' ? 'Spendenseite' : 'Betriebsseite'}}
+            </h1>
+
+        </div>
+        <v-container>
+            <v-row>
+                <v-col>
+                    <v-card class="projectBox" elevation="4">
+                        <v-card-title class="font-weight-light display-1">
+                            {{project.name}}
+                        </v-card-title>
+                        <v-card-text>
+                            <div class="mb-4">
+                                <h3>Adresse</h3>
+                                {{project.street}}<br>
+                                {{project.zip}} {{project.city}}
+                            </div>
+                            <v-card elevation="7" class="py-6 text-center">
+                                <v-row>
+                                    <v-col>
+                                        <h4 class="headline">Gesammelt</h4>
+                                        <h1 class="display-2 font-weight-thin">{{project.donationGoal.reached.toFixed(2)}}€</h1>
+
+                                    </v-col>
+                                    <v-col>
+                                        <h4 class="headline">Ziel</h4>
+                                        <h1 class="display-2 font-weight-thin">{{project.donationGoal.total.toFixed(2)}}€</h1>
+
+                                    </v-col>
+                                </v-row>
+                                <h2>{{(project.donationGoal.reached/project.donationGoal.total*100).toFixed(1)}}%</h2>
```

Comment:
computed property? dann ist klarer was gemeint ist mit nem vernünftigen namen.
vorallem benutzt du das  `project.donationGoal.reached/project.donationGoal.total*100` in zeile 42 nochmal

Reply:
Thanks! Sollte zu Ihrer Zufriedenheit behoben worden sein tun.